README.md

```diff
@@ -1,57 +1,8 @@
-# Northcoders House of Games API
+**please follow the link for gitHub repository https://github.com/JakeChristie91/jake-christie-nc-games;
 
-## Background
+**type ----> git clone https://github.com/JakeChristie91/jake-christie-nc-games <------ into your computer terminal and hit enter;
 
-We will be building an API for the purpose of accessing application data programmatically. The intention here is to mimic the building of a real world backend service (such as reddit) which should provide this information to the front end architecture.
+**install express & supertest
 
-Your database will be PSQL, and you will interact with it using [node-postgres](https://node-postgres.com/).
+**Note. You'll need to run npm install & npm innit
 
```

Comment:
You should mention the .env files (test and development) that you need to make in order to make the databases